### ISSUE TITLE
Enable Azure Policy Add-On on apps AKS cluster

### ISSUE THREAD

**Issue by bengtfredh** [Altinn/altinn-platform#270]
### Description

For more details please visit [Understand Azure Policy for Azure Kubernetes Service](https://docs.microsoft.com/en-ie/azure/governance/policy/concepts/rego-for-aks)

**Comment by annerisbakk** [comment]
@bengtfredh Noe vi bør ta tak i?

**Comment by bengtfredh** [comment]
@annerisbakk Yes, we should look in to this. It will become useful when we start work on policies in Azure (enabling team)

**Comment by Herskis** [comment]
@bengtfredh Noe vi egentlig bør ta opp i møtet med sikkerhet som ønsker Defender slått på for Kubernetes?

**Comment by bengtfredh** [comment]
Dette er noe annet enn defender - vi har valgt Kyverno fremfor azure policy.
Kan lukkes
